common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -75,8 +75,8 @@ object ActivationMessage extends DefaultJsonProtocol {
  * Message that is sent from the invoker to the controller after action is completed or after slot is free again for
  * new actions.
  */
-abstract class AcknowledegmentMessage() extends Message {
-  override val transid: TransactionId
+abstract class AcknowledegmentMessage(private val tid: TransactionId) extends Message {
```

Comment:
Had to change code here to workaround Scala compiler bug (see previous comments)